plugins/base/src/test/resources/expect/signatureTest/out/images/arrow_down.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="10" height="7" viewBox="0 0 10 7" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Are you sure this is signature specific file? If it does not, it should definitely be moved out of this PR